css/typography.css

```diff
@@ -62,15 +62,21 @@ h1, .h1 {
 }
 
 h2, .h2 {
-    font-size: 3.2rem;
+    font-size: 2.8rem;
     font-weight: var(--font-weight-extrabold);
     line-height: 1.2;
+    margin: 1.5em 0 0.5em;
+}
+
+h2:first-of-type, .h2:first-of-type {
+    margin: 0.75em 0 0.5em;
```

Comment:
Having this act differently to everything else looks a bit weird:

<img width="389" height="562" alt="image" src="https://github.com/user-attachments/assets/49ac8202-bb48-46ad-bdb7-78c69b865b3b" />

There's not enough detail in https://github.com/silverstripe/startup-theme/issues/23 for me to know what the intention was - is there a design document that shows the first h2 being different from other h2 and not being that way for any other heading type? Or some explanation of why h2 gets such a massive gap (except for the first one)?